src/components/Profile/Profile.scss

```diff
@@ -47,38 +56,43 @@
       text-align: left;
     }
   }
-  .personal__info .uplode-img {
+  .personal__info .upload-img {
     position: relative;
   }
   .img_profile {
     width:50%
   }
-  .personal-info__uplode label {
-
-    display: flex;
-    justify-content: center;
-    position: absolute;
+  .upload-btn{
+    padding: 20px;
+    position: relative;
+  }
+  .avatar-section__upload label {
+    padding: 8px 20px;
+    border-radius: 20px;
     background-color: #f8f8f8;
-    padding: 10px;
-    border: 1px solid rgba(0,0,0,0.1);
-    color: #9aa4c8;
-    font-size: calc(13px + (15 - 13) * ((100vw - 320px) / (1920 - 320)));
-    cursor: pointer;
+    background-color: $primary-color;
+    font-size: 1.4rem;
+    font-weight: 700;
+    color: $white;
   }
-  .uplode-img {
+  .upload-img {
     display: flex;
     align-items: center;
     flex-direction: column;
     @media screen and (min-width: 576px) {
       align-items: flex-start;
     }
   }
-  .uplode-img input {
-    background-color: #f8f8f8;
-    border: 1px solid rgba(0,0,0,0.1);
-    color: #9aa4c8;
-    font-size: calc(13px + (15 - 13) * ((100vw - 320px) / (1920 - 320)));
+  .upload-img input {
+    width: 0;
+    height: 0;
     opacity: 0;
+    position: absolute;
+    // background-color: #f8f8f8;
+    // border: 1px solid rgba(0,0,0,0.1);
+    // color: #9aa4c8;
+    // font-size: calc(13px + (15 - 13) * ((100vw - 320px) / (1920 - 320)));
+    // opacity: 0;
```

Comment:
just remove this commented code in a next PR or commit.

Reply:
Done!